docs/website/blog/2022-09-22-odo-v3-rc2.md

```diff
@@ -0,0 +1,94 @@
+---
+title: odo v3.0.0-rc2 Released
+author: Dharmit Shah
+author_url: https://github.com/dharmit
+author_image_url: https://github.com/dharmit.png
+tags: ["release"]
+slug: odo-v3-rc2-release
+---
+`3.0.0-rc2` of odo has been released!
+
+<!--truncate-->
+
+To install `odo`, follow our installation guide at [odo.dev](../docs/overview/installation)
+
+## Note about Release Candidate
+
+With rc2, we are feature complete. Our GA release will include:
+- Onboarding updates, including updates to the Quickstart guide
+- Video tutorial
+
+## Notable Changes
+
+Check this [YouTube playlist](https://www.youtube.com/watch?v=PgDsXlvb5MQ&list=PLGMB2PY4SNOqz4sAwtNOpEvb_ZYkiUfAz) 
+for an overview of the most notable changes in this release.
+
+### Features
+
+#### `odo dev` manually sync files upon pressing "p" ([#6089](https://github.com/redhat-developer/odo/pull/6089))
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/uTV8dlflZHs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+`odo dev --no-watch` can now sync files when user presses the key "p".
```

Comment:
```suggestion
`odo dev` can now sync files when user presses the key "p".
```

To me, this is applicable to all the `odo dev` command, regardless of the `no-watch` flag.